### ISSUE TITLE
Issue when running Pool fucntions "jq: error: Could not open file /tmp/cntools/ledger-state.json: No such file or directory"

### ISSUE THREAD

**Issue by hotlimbo** [cardano-community/guild-operators#402]
I have this issue which rose on one of my pools yesterday. Please see the attached screenshots.
These occur when operating the Pool function in CNTools. This occurred yesterday in version 5.2 and the newer 5.3 version today.

On my second pool I tested this function yesterday and it worked fine. However, today when I returned to my second node server the problem surface there too. Now I see on the telegram chat that there are others having the same issue.

![Annotation 2020-08-02 095916](https://user-images.githubusercontent.com/14864253/89234959-239de980-d5e5-11ea-808b-c220f9d36df8.jpg)
![Annotation 2020-08-02 100013](https://user-images.githubusercontent.com/14864253/89234961-24368000-d5e5-11ea-81fa-504f9e9338e8.jpg)
![Annotation 2020-08-02 140922](https://user-images.githubusercontent.com/14864253/89234963-24cf1680-d5e5-11ea-885e-f576183c58c2.jpg)
![Annotation 2020-08-02 141531](https://user-images.githubusercontent.com/14864253/89234965-24cf1680-d5e5-11ea-9b9d-443754323a44.jpg)

**To Reproduce**
Basically using the "Pool" choice in the CNTools.sh script

**Expected behavior**
Should be able to read and execute pool functions

**Screenshots**

**Version:**
 - OS: [e.g. UBUNTU 20.4]
 - Product version: [e.g. CNTools 5.3]
 - Cardano Node version: 1.18
 - Network you're connecting to: [output of `jq -r .systemStart $CNODE_HOME/files/genesis.json`]

**Comment by hotlimbo** [comment]
We added this line at the end of cntools.config:

`TIMEOUT_LEDGER_STATE=300`

And it seemed to resolve the issue. But still, maybe check up on this?

**Comment by rdlrt** [comment]
@hotlimbo - Changelog prompted you to run prereqs.sh, not doing so will lead to that issue

**Comment by michaeljohngoff** [comment]
> 
![photo_2020-08-05_03-18-44](https://user-images.githubusercontent.com/69106762/89370593-09453800-d6d9-11ea-851a-c4a3b09a1682.jpg)
